### PR TITLE
Gold pyramid on map

### PR DESCRIPTION
**Pull request by Vgoloshivskiy**
#### Summary
Content "Adds a bit more Yrax content"

#### Purpose of change
adds a bit more yrax content

#### Describe the solution
add more yrax content

#### Testing
spawned few times and used emp ->didn't throw errors

#### Additional context
draft for now as i don't like the way mapgenerator looks

### CODE REVIEW

File: data/json/overmap/map_extras.json [hunk 1]
```diff
@@ -214,6 +214,18 @@
     "autonote": true,
     "flags": [ "YRAX" ]
   },
+  {
+    "id": "mx_yrax_pyramid_cage",
+    "type": "map_extra",
+    "name": { "str": "Golden pyramid" },
+    "description": "Here is golden pyramid with asymetricals patterns carven in it.",
```
Comment (Qrox) [inline data/json/overmap/map_extras.json:221]:
```suggestion
    "description": "Here is a golden pyramid with asymmetric patterns carven in it.",
```

**Comment by Naadn** [comment]
I don't think put `mon_lieutenant_shadow` in that cage is a good idea.

**Comment by Vgoloshivskiy** [comment]
> I don't think put `mon_lieutenant_shadow` in that cage is a good idea.

Why? it is more of a gamble -> good event/bad event/neutral event(lore)

**Comment by Naadn** [comment]
> > I don't think put `mon_lieutenant_shadow` in that cage is a good idea.
> 
> Why? it is more of a gamble -> good event/bad event/neutral event(lore)
Isn't `mon_lieutenant_shadow` have a very complex EoC that it will just disappear from that cage?

**Comment by Vgoloshivskiy** [comment]
> > > I don't think put `mon_lieutenant_shadow` in that cage is a good idea.
> > 
> > 
> > Why? it is more of a gamble -> good event/bad event/neutral event(lore)
> 
> Isn't `mon_lieutenant_shadow` have a very complex EoC that it will just disappear from that cage?

0 idea it will spawn only after you hit it with emp and at that point you can get inside pyramid and it can get outside no problem, before emp hit the cage it is empty

**Comment by harakka** [comment]
The shadow blob lieutenant is a unique special event mob, it doesn't belong here. Does the effect really need c++? This is kinda specific thing to live in hardcode land.

**Comment by Vgoloshivskiy** [comment]
> The shadow blob lieutenant is a unique special event mob, it doesn't belong here. Does the effect really need c++? This is kinda specific thing to live in hardcode land.

k i ll remove him later, I have no idea how to add eoc for terrain being hit by emp so did it with c++

**Pull request closed without merging**